can't run translations example

```
[user@~/dir/translations]$ npm install
[user@~/dir/translations]$ npm run start

> react-intl-example-translations@1.0.0 start /Users/user/dir/translations
> cross-env ../../node_modules/.bin/babel-node src/server/index.js

sh: cross-env: command not found

npm ERR! Darwin 15.3.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "start"
npm ERR! node v6.9.2
npm ERR! npm  v4.0.3
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! react-intl-example-translations@1.0.0 start: `cross-env ../../node_modules/.bin/babel-node src/server/index.js`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the react-intl-example-translations@1.0.0 start script 'cross-env ../../node_modules/.bin/babel-node src/server/index.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the react-intl-example-translations package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     cross-env ../../node_modules/.bin/babel-node src/server/index.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs react-intl-example-translations
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls react-intl-example-translations
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/user/dir/translations/npm-debug.log
```

installing cross-env then running:
```
[user@~/dir/translations]$ sudo npm install -g cross-env
Password:
/usr/local/bin/cross-env -> /usr/local/lib/node_modules/cross-env/bin/cross-env.js
/usr/local/lib
└─┬ cross-env@3.1.4 
  └─┬ cross-spawn@3.0.1 
    ├─┬ lru-cache@4.0.2 
    │ ├── pseudomap@1.0.2 
    │ └── yallist@2.0.0 
    └─┬ which@1.2.12 
      └── isexe@1.1.2 

[user@~/dir/translations]$ npm run start

> react-intl-example-translations@1.0.0 start /Users/user/dir/translations
> cross-env ../../node_modules/.bin/babel-node src/server/index.js

events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: spawn ../../node_modules/.bin/babel-node ENOENT
    at exports._errnoException (util.js:1022:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:193:32)
    at onErrorNT (internal/child_process.js:359:16)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
    at Module.runMain (module.js:606:11)
    at run (bootstrap_node.js:394:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:509:3

npm ERR! Darwin 15.3.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "start"
npm ERR! node v6.9.2
npm ERR! npm  v4.0.3
npm ERR! code ELIFECYCLE
npm ERR! react-intl-example-translations@1.0.0 start: `cross-env ../../node_modules/.bin/babel-node src/server/index.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the react-intl-example-translations@1.0.0 start script 'cross-env ../../node_modules/.bin/babel-node src/server/index.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the react-intl-example-translations package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     cross-env ../../node_modules/.bin/babel-node src/server/index.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs react-intl-example-translations
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls react-intl-example-translations
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/user/dir/translations/npm-debug.log
```

--- a/examples/async/package.json
+++ b/examples/async/package.json
@@ -5,10 +5,10 @@
   "private": true,
   "main": "index.js",
   "scripts": {
-    "clean": "cross-env ../../node_modules/.bin/rimraf build",
-    "build": "cross-env ../../node_modules/.bin/mkdirp build && cross-env ../../node_modules/.bin/browserify index.js -d -t babelify -t browserify-shim > build/bundle.js",
-    "watch": "cross-env ../../node_modules/.bin/mkdirp build && cross-env ../../node_modules/.bin/watchify index.js -d -t babelify -t browserify-shim -o build/bundle.js -v",
-    "start": "cross-env ../../node_modules/.bin/babel-node server.js"
+    "clean": "../../node_modules/.bin/cross-env ../../node_modules/.bin/rimraf build",
+    "build": "../../node_modules/.bin/cross-env ../../node_modules/.bin/mkdirp build && ../../node_modules/.bin/cross-env ../../node_modules/.bin/browserify index.js -d -t babelify -t browserify-shim > build/bundle.js",
+    "watch": "../../node_modules/.bin/cross-env ../../node_modules/.bin/mkdirp build && ../../node_modules/.bin/cross-env ../../node_modules/.bin/watchify index.js -d -t babelify -t browserify-shim -o build/bundle.js -v",
+    "start": "../../node_modules/.bin/cross-env ../../node_modules/.bin/babel-node server.js"
   },
   "author": "James Kyle <me@thejameskyle.com>",
   "license": "BSD-3-Clause",

--- a/examples/nested/package.json
+++ b/examples/nested/package.json
@@ -5,13 +5,13 @@
   "private": true,
   "main": "index.js",
   "scripts": {
-    "clean": "cross-env ../../node_modules/.bin/rimraf build",
-    "build:bundle": "cross-env ../../node_modules/.bin/mkdirp build && cross-env ../../node_modules/.bin/browserify src/client/index.js -d -t babelify -t browserify-shim > build/bundle.js",
-    "build:group-messages": "cross-env ../../node_modules/.bin/babel-node scripts/group-messages.js",
+    "clean": "../../node_modules/.bin/cross-env ../../node_modules/.bin/rimraf build",
+    "build:bundle": "../../node_modules/.bin/cross-env ../../node_modules/.bin/mkdirp build && ../../node_modules/.bin/cross-env ../../node_modules/.bin/browserify src/client/index.js -d -t babelify -t browserify-shim > build/bundle.js",
+    "build:group-messages": "../../node_modules/.bin/cross-env ../../node_modules/.bin/babel-node scripts/group-messages.js",
     "build": "npm run build:bundle && npm run build:group-messages",
-    "watch:bundle": "cross-env ../../node_modules/.bin/mkdirp build && cross-env ../../node_modules/.bin/watchify src/client/index.js -d -t babelify -t browserify-shim -o build/bundle.js -v",
+    "watch:bundle": "../../node_modules/.bin/cross-env ../../node_modules/.bin/mkdirp build && ../../node_modules/.bin/cross-env ../../node_modules/.bin/watchify src/client/index.js -d -t babelify -t browserify-shim -o build/bundle.js -v",
     "watch": "npm run build:group-messages && npm run watch:bundle",
-    "start": "cross-env ../../node_modules/.bin/babel-node src/server/index.js"
+    "start": "../../node_modules/.bin/cross-env ../../node_modules/.bin/babel-node src/server/index.js"
   },
   "author": "Eric Ferraiuolo <edf@ericf.me>",
   "license": "BSD-3-Clause",

--- a/examples/translations/package.json
+++ b/examples/translations/package.json
@@ -5,13 +5,13 @@
   "private": true,
   "main": "index.js",
   "scripts": {
-    "clean": "cross-env ../../node_modules/.bin/rimraf build",
-    "build:bundle": "cross-env ../../node_modules/.bin/mkdirp build && cross-env ../../node_modules/.bin/browserify src/client/index.js -d -t babelify -t browserify-shim > build/bundle.js",
-    "build:langs": "cross-env ../../node_modules/.bin/babel-node scripts/translate.js",
+    "clean": "../../node_modules/.bin/cross-env ../../node_modules/.bin/rimraf build",
+    "build:bundle": "../../node_modules/.bin/cross-env ../../node_modules/.bin/mkdirp build && ../../node_modules/.bin/cross-env ../../node_modules/.bin/browserify src/client/index.js -d -t babelify -t browserify-shim > build/bundle.js",
+    "build:langs": "../../node_modules/.bin/cross-env ../../node_modules/.bin/babel-node scripts/translate.js",
     "build": "npm run build:bundle && npm run build:langs",
-    "watch:bundle": "cross-env ../../node_modules/.bin/mkdirp build && cross-env ../../node_modules/.bin/watchify src/client/index.js -d -t babelify -t browserify-shim -o build/bundle.js -v",
+    "watch:bundle": "../../node_modules/.bin/cross-env ../../node_modules/.bin/mkdirp build && ../../node_modules/.bin/cross-env ../../node_modules/.bin/watchify src/client/index.js -d -t babelify -t browserify-shim -o build/bundle.js -v",
     "watch": "npm run build:langs && npm run watch:bundle",
-    "start": "cross-env ../../node_modules/.bin/babel-node src/server/index.js"
+    "start": "../../node_modules/.bin/cross-env ../../node_modules/.bin/babel-node src/server/index.js"
   },
   "author": "Eric Ferraiuolo <edf@ericf.me>",
   "license": "BSD-3-Clause",

